Makefile

```diff
@@ -859,10 +869,10 @@ benchmark: all
 	@$(PONY_BUILD_DIR)/libponyrt.benchmarks
 
 stdlib-debug: all
-	$(PONY_BUILD_DIR)/ponyc -d --checktree --verify packages/stdlib
+	$(PONY_BUILD_DIR)/ponyc $(OPENSSL) -d --checktree --verify packages/stdlib
```

Comment:
The `$(OPENSSL)` argument to `ponyc` should not longer be required due to the use of `default_openssl`.

Reply:
The use of default_openssl is optional so i believe these are still needed.

Reply:
I lean towards recommending people use `default_openssl` similar to `default_pic` and removing these extra arguments but I don't feel very strongly about it.

I leave it to you and other other folks who decide to weigh in to make a decision about what to do.

Reply:
I agree, its not needed if you assume you don't want change the value when using the Makefile. But one use I had was to use the Makefile to build the compiler and with a target of test-stdlib using default_openssl=openssl_1.1.0. This would compile the compiler and stdlib and then run it. I can then use the Makefile again to compile only stdlib using OPENSSL=-Dopenssl_0.9.0 to test that stdlib would not compile.

Probably not a normal case, but I have used it. I don't have a strong opinion either, but given that this is working and has some uses I'm inclined to leave.